tests/test_other.py

```diff
@@ -666,6 +666,17 @@ def test_cmake_compile_features(self):
     emscripten_features = '\n'.join([x for x in emscripten_features.split('\n') if '***' in x])
     self.assertTextDataIdentical(native_features, emscripten_features)
 
+  # Test that the user's explicitly specified generator is always honored
+  # Internally we override the generator on windows, unles the user specifies one
+  # Test require Ninja to be installed
+  def test_cmake_explicit_generator(self):
+    if not utils.which('ninja'):
```

Comment:
Does this test actually need/use ninja?    Isn't it just testing the generation of the ninja file itself?

Reply:
Yes it didn't use Ninja itself. But I guess we won't need this test if ninja is not an available choice?

Reply:
By the way @sbc100 Sam, do you know why the CI check "test-browser-chrome" failed? I've got no idea 😕

Reply:
Almost certainly a flake I can re-run if it happens again.

If you fix the flake8 issue and re-upload I think that should be enough to fix it.